base/_mixins.scss

```diff
@@ -1,50 +1,97 @@
-// @file mixins.scss
+////
+// Oomph Scaffold Mixins
+// @access public
+// @group Oomph Mixins
+// @author Oomph, Inc. UX Engineers
+// @content the @content directive is not used
+// @parameter none
+// @require no 3rd party requirements
+////
 
-// @link http://csswizardry.com/2016/02/mixins-better-for-performance/
 
-// Provide an accesible hover for non-touch devices, turns it into the active state for mobile, and maintain a no-js fallback
-// @requires Modernizr as a JS dependency to get .no-touch classes
-// Adds styles via @content
-// @jhogue
+// Default constants used by these mixins
+// Change below only if you require a different value
+// These are set by Bourbon
+// $em-base: 16px;
+$ICON_FONT_FAMILY: 'font-awesome';
 
-// Usage:
-// .link {
-//   padding: .5em 1em; // Styles that do not change when interacting with this element
-//
-//   @include touch-hover('idle') {
-//     color: black;
-//   }
-//   @include touch-hover('hover') {
-//     color: red;
-//   }
-//
-//   &__disabled {
-//     @include touch-hover('hover', true) {
-//      color: silver;
-//     }
-//   }
-// }
 
-// Outputs:
-// .link {
-//   padding: .5em 1em;
-// }
-// .link, .link:link, .link:visited {
-//   color: black;
-// }
-// .no-js .link:hover, .no-js .link:focus, .js.no-touchevents .link:hover, .js.no-touchevents .link:focus, .js.touchevents .link:active {
-//   color: red;
-// }
-// .no-js .link__disabled, .no-js .link__disabled:link, .no-js .link__disabled:visited, .no-js .link__disabled:hover, .no-js .link__disabled:focus, .js.no-touchevents .link__disabled, .js.no-touchevents .link__disabled:link, .js.no-touchevents .link__disabled:visited, .js.no-touchevents .link__disabled:hover, .js.no-touchevents .link__disabled:focus, .js.touchevents .link__disabled, .js.touchevents .link__disabled:active {
-//   cursor: default;
-//   color: silver;
-// }
+/// touch-hover() =
+/// Provide an accesible hover for non-touch devices, turns it into the active state for mobile, and maintains a no-js fallback
+/// @author jhogue
+/// @require Modernizr.js to get .no-touchevents classes
+/// @content the @content directive is used
```

Comment:
For touch-hover, @content is used. That is how styles get passed through the mixin.